README.md

```diff
@@ -1,3 +1,53 @@
-# Camunda FEEL builtins
 
-Collection of builtin camunda extensions for FEEL
\ No newline at end of file
+# Camunda FEEL Builtins
+
+Collection of builtin Camunda extensions for FEEL (Friendly Enough Expression Language). These builtins get extracted from [camunda-docs](https://github.com/camunda/camunda-docs/tree/main/docs/components/modeler/feel/builtin-functions).
+
+## Installation
+
+Install via npm:
+
+```sh
+npm install @camunda/feel-builtins
+```
+
+## Usage
+
+You can use the builtins in your FEEL editor by importing them from this package. Here's an example of how to integrate them into a FEEL editor instance:
```

Comment:
Link `feel-editor`, consider integration with linting.

```suggestion
In your [FEEL editor](https://github.com/bpmn-io/feel-editor) you can use these builtins to establish the Camunda context:
```

Consider: Should this example also feature the Camunda dialect? I think that would make sense.

Consider: Should we also feature integration with feel-lint? I think that would make sense.